app/controllers/users_controller.rb

```diff
@@ -49,7 +49,7 @@ def create
 
   def update
     respond_to do |format|
-      if @user.update_attributes(user_params)
+      if @user.update_attributes(user_params.except(User.non_updateable_params))
```

Comment:
@carpodaster 
i've added disabled: true to the form in edit user template, so user can't access it  - shall i leave it that way or let it display error when changed